.bazelproject

```diff
@@ -22,3 +22,6 @@ workspace_type: java
 
 build_flags:
   --incompatible_depset_is_not_iterable=false
+
+test_flags:
```

Comment:
Can you revert this? It would be useful for Travis, but is annoying for local development